libs/platform/experience/src/services/experience/default-experience.service.ts

```diff
@@ -104,14 +104,22 @@ export class DefaultExperienceService implements ExperienceService {
   }
 
   protected reloadComponent(uid: string): void {
+    /**
+     * @deprecated Since version 1.4. Use provided `ExperienceAdapter.get` method.
+     */
```

Comment:
implementation detail, hidden in the method, why do we deprecate?

Reply:
It's protected method so available for user. And also user might not have injected `ExperienceAdapter` which we introduced on 1.1 version so the have fallback for old http server url

Reply:
But that i'd expect it to be on the header of the method